NitroxClient/GameLogic/InitialSync/StoryGoalInitialSyncProcessor.cs

```diff
@@ -44,11 +44,7 @@ private void SetRadioQueue(List<string> radioQueue)
         private void SetCompletedStoryGoals(List<string> storyGoalData)
         {
             StoryGoalManager.main.completedGoals.Clear();
-
-            foreach (string completedGoal in storyGoalData)
-            {
-                StoryGoalManager.main.completedGoals.Add(completedGoal);
-            }
+            StoryGoalManager.main.completedGoals.AddRange(storyGoalData);
```

Comment:
Is this always acessible or the singleton is prone to errors ?

Reply:
The StoryGoalManager.main singleton is never null at this moment since its Awake method happens during the base game loading and the InitialSync happens after.